gamemodes/terrortown/gamemode/shared/sh_sql.lua

```diff
@@ -233,11 +233,26 @@ end
 -- @param table keys the keys for the data @{table}
 -- @return boolean Whether the database table was created successfully
 -- @realm shared
--- @todo usage
+-- @deprecated
 function sql.CreateSqlTable(tableName, keys)
-    local result
-
-    if not sql.TableExists(tableName) then
+    Dev(
+        2,
+        "`sql.CreateSqlTable` is deprecated and should be handled by migrations.",
+        "This ran for table:",
+        tableName
+    )
+    if sql.TableExists(tableName) then
+        Dev(
+            2,
+            "The database table",
+            tableName,
+            "already exists.",
+            "Adding columns to an existing database table via `sql.CreateSqlTable` was removed."
+        )
+        -- We need to return true here to not break our current assumptions
+        -- in our current codebase (database library, roleselection, huds)
+        return true
```

Comment:
Maybe we should check keys and return false if somebody tries to change an existing database.

Reply:
this would be somewhat of a catch-22 with the `pragma` problem :D given that we would need to check the existing keys of the database table

Reply:
Damn, I see. Didnt know that.